code/modules/mob/new_player/new_player.dm

```diff
@@ -107,7 +107,7 @@
 
 		if(href_list["observe"])
 
-			if(alert(src,"Are you sure you wish to observe? You will have to wait 30 minutes before being able to respawn!","Player Setup","Yes","No") == "Yes")
+			if(alert(src,"Are you sure you wish to observe? You will have to wait 20 minutes before being able to respawn!","Player Setup","Yes","No") == "Yes")

```

Comment:
you var'd it, so might as well replace it with respawntime/60 to save on editing those later